pkg/compiler/analysis.go

```diff
@@ -17,6 +17,7 @@ var (
 		"AppCall",
 		"FromAddress", "Equals",
 		"panic",
+		"ToBool", "ToByteArray", "ToInteger",
```

Comment:
Am I correct that in fact plain `ToBool()` invocation would comple the same way `convert.ToBool()` does?

Reply:
Yes, this is a known problem with current interop implementation.

Reply:
Do we have an issue for it?